packages/amplication-server/src/core/build/build.service.ts

```diff
@@ -2,7 +2,7 @@ import { Inject, Injectable, forwardRef } from "@nestjs/common";
 import { ConfigService } from "@nestjs/config";
 import { Prisma, PrismaService } from "../../prisma";
 import { LEVEL, MESSAGE, SPLAT } from "triple-beam";
```

Comment:
@manulpatel you need to remove the unused import to pass the lint checks. Can you please address this?